cmd/tendermint/commands/run_node.go

```diff
@@ -35,27 +35,27 @@ var (
 func init() {
 
 	// configuration options
-	runNodeCmd.Flags().StringVar(&moniker, "moniker", config.GetString("moniker"),
```

Comment:
Given that path to config is now dynamic (`--data path`; `path/config.toml`), I don't see a way to read it beforehand and display default values cause we don't know the path to config, doh.

Maybe we could create an instance of default Config in memory and get defaults from it.  How does that sound? Or maybe there is a way to solve this puzzle using Viper?